packages/front-end/pages/experiments/index.tsx

```diff
@@ -362,9 +361,8 @@ const ExperimentsPage = (): React.ReactElement => {
                     <td className="nowrap" title={datetime(e.date)}>
                       {ago(e.date)}
                     </td>
-                    {tab === "stopped" && (
+                    {tab === "stopped" && e.results && (
```

Comment:
I think this will break rendering of the table.  If the tab is `stopped`, we always want to render a `<td>` here (so all rows have the same columns).  We should move the `e.results` check to the contents of the `<td>` instead.

Reply:
Good catch, I made that change here 337fd50b293b79bb8ff0670c58e4535e4195b242